_posts/2017/2017-06-13-reactive-streams.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: post
+title: "Reactive streams"
+date: "2017-06-13 08:00:00"
+image: blog-header/api-journey.jpg
+categories: tech-stories
+authors: ["Christian"]
+---
+
+# akka-streams
```

Comment:
Is this the sub-headline after the title? Or is this somehow redundant with the title?